include/bout/mesh.hxx

```diff
@@ -730,7 +759,7 @@ class Mesh {
 
 private:
   /// Allocates a default Coordinates object
-  Coordinates *createDefaultCoordinates();
+  Coordinates *createDefaultCoordinates(const CELL_LOC location);
```

Comment:
Is it worth setting a default location so existing calls still work and do the same thing?

Reply:
It's a private method only used by Mesh::coordinates() so I don't think there should be any other existing calls.